chiron/states.py

```diff
@@ -22,6 +23,7 @@ class SamplerState:
     def __init__(
         self,
         x0: unit.Quantity,
+        random_seed: random.PRNGKey,
```

Comment:
random.PRNGKey is the current state of the sampler...hence I don't think `random_seed` is the current name. This should probably be called `current_key` or `current_PRNG_key`.  

I think the sampler state `init`  function should accept two things:
- random_seed that would be used to generate the first key. 
- current_key which would just set self._current_key. 
where if current_key is defined, we will ignore the random_seed if set, or throw an error.

Reply:
According to https://jax.readthedocs.io/en/latest/jax-101/05-random-numbers.html, `seed` and `key` are synonyms :-)  
But, I agree that it is clearer if we call it `current_PRNG_key`.

I would prefer to only pass `current_PRNG_key` and use it to generate the first key of the new stream. Is there a use case in which we would like to manually set the `current_key`?